csrc/runtime/fusion_cache_utils.cpp

```diff
@@ -66,10 +68,94 @@ KernelArgumentHolder ArgumentManager::translateValsToArgs(
   return holder;
 }
 
+// When a fusion segment is executed by ExprEvalExecutor, the output
+// sizes and strides are determined by PyTorch, which might be different from
+// how we infer them. In this case, PyTorch is the ground truth, so we should
+// use PyTorch's result to reset the allocation domain and contiguity.
+void resetAllocationDomainAndContiguity(
```

Comment:
Is this trying to hide some bugs? I understand that PyTorch is the ground truth, but it sounds like a bug when our allocation isn't inconsistent. For example, AliasAnalysis predicts what ATen ops would generate and marks an operation as meta [only if](https://github.com/NVIDIA/Fuser/blob/524dcf12dba1a4dc1318a1bcc60b6e45ff4c8d3e/csrc/alias_analysis.cpp#L78) the allocation domain is consistent.